recipes/seiqrdp-model/meta.yaml

```
@@ -0,0 +1,52 @@
+package:
+  name: seiqrdp-model
+  version: 7.5.3
+
+source:
+  url: https://pypi.io/packages/source/s/seiqrdp-model/seiqrdp_model-7.5.3.tar.gz
+  sha256: 8ccdeee2351f79e787ef8e0ec2d88b24ecedfc37767214a1383a38e2fcbdd0f6
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - colorama
+    - matplotlib
```

Does this need the `qt` frontend for matplotlib? If not:
```suggestion
    - matplotlib-base
```